src/draft/Person/JobApplicantProfile.py

```diff
@@ -0,0 +1,4757 @@
+        example="http://data.europa.eu/esco/skill/6d3edede-8951-4621-a835-e04323300fa0",
+    )
+    skill_level: CERFLevel = Field(
+        None,
+        title="Skill level",
+        description="The language skill level according to Common European Framework "
+        "of Reference for Languages, known as CERF.",
+        example=CERFLevel.B2,
+    )
+
+
+class OtherSkill(CamelCaseModel):
+    esco_identifier: Optional[HttpUrl] = Field(
+        None,
+        title="ESCO Identifier",
+        description="The skill identifier based on European Standard Classification of "
+        "Occupations (ESCO).",
+        example="http://data.europa.eu/esco/skill/869fc2ce-478f-4420-8766-e1f02cec4fb2",
+    )
+    skill_level: SkillLevel = Field(
+        None,
+        title="Skill level",
+        description="The language skill level according to Common European Framework "
+        "of Reference for Languages, known as CERF.",
+        example=SkillLevel.BEGINNER,
+    )
+
+
+class Certification(CamelCaseModel):
+    certification_name: str = Field(
+        None,
+        title="Certification name",
+        description="The name of the acquired qualification.",
+        example="Hygiene passport",
+        max_length=250,
+    )
+    qualification_type: str = Field(
+        None,
+        title="Qualification type",
+        description="Type of qualification clarifies how the qualification has "
+        "been acquired (e.g. Self-organized / Institutional).",
+        example="Institutional",
+        max_length=250,
+    )
+
+
+class Permit(CamelCaseModel):
+    permit_name: str = Field(
+        None,
+        title="Permit name",
+        description="The name of the acquired permit.",
+        example="Hot work permit",
+        max_length=250,
+    )
+    permit_type: str = Field(
+        None,
+        title="Permit type",
+        description="Type of qualification clarifies how the qualification has been "
+        "acquired (e.g. Self-organized / Institutional).",
+        example="Institutional",
+        max_length=250,
+    )
+
+
+class WorkPreference(CamelCaseModel):
+    preferred_region: List[ISO_3166_2_FI] = Field(
+        None,
+        title="Preferred region",
+        description="The potential regions in Finland that the user desired to find a "
+        "job from.",
+        example=ISO_3166_2_FI.FI_18,
+    )
+    preferred_municipality: List[FinnishMunicipality] = Field(
+        None,
+        title="Preferred municipality",
+        description="The potential municipalities in Finland that the user desires to "
+        "find a job from.",
+        example=FinnishMunicipality.HELSINKI,
+    )
+    type_of_employment: EmploymentType = Field(
+        None,
+        title="Type of employment",
+        description="The type of employment contract that the person is looking for.",
+        example=EmploymentType.PERMANENT,
+    )
+    working_time: WorkingTime = Field(
```

Comment:
Add fields for the working time property as defined in https://koodistot.suomi.fi/codescheme;registryCode=dataecon;schemeCode=work_time